src/lib/logs/dynatrace_client.py

```diff
@@ -43,6 +47,7 @@ def send_logs(context: LogsContext, logs: List[Dict]):
             encoded_body_bytes = batch.encode("UTF-8")
             context.log('Log ingest payload size: {} kB'.format(round((len(encoded_body_bytes) / 1024), 3)))
 
+            self_monitoring.all_requests += 1
```

Comment:
IMO we are missing reporting of size of sent ingest (the number we are printing in line 48) and optionally the number of sent log entries, although this one can be inferred from number of ackes pubsub messages.

Reply:
Added both, however calculating number of sent logs entries need to be improved after improvement for ack messages (to not acknowledge messages when request to Dynatrace fails)